### PR TITLE
Add fps_lock method to GameLoop and public attr fps_target

### LINKED ISSUE #26: Add FPS lock mechanism
That should be possible to be setup directly on the current examples. Maybe through a class attribute like `targetFPS` with a integer value.

### PR DESCRIPTION
**Pull request by ryukinix**
Closes #26 

Set fps_target for all examples

- lerax-breakout -> 60fps
- lerax-square -> 60fps
- lerax-square-platform -> 120fps (60fps is not enough)
- wesleycsj-doom-fire -> 60fps
- wesleycsj-hello-world -> 60fps

Now the CPU consumption is very low! My CPU temp is not burning
anymore.

### CODE REVIEW

File: examples/wesleycsj-hello-world.cpp [hunk 1]
```diff
@@ -61,5 +61,6 @@ class Game: public GameLoop {
 
 int main(int argc, char** argv){
 	Game game("Hello World", SCREEN_WIDTH, SCREEN_HEIGHT);
+    game.fps_target = 60;
```
Comment (ryukinix) [inline examples/wesleycsj-hello-world.cpp:64]: I need to remove the tabs of that file.

**Comment by ryukinix** [comment]
> Ok, the functionality seems work well. The only thing, less important, is the name of the variable, i don't think which fps_target is a good name for variable, and doesn't better to it be setted in something like a set method? Like this, "game.setMaxFramerate(60);" ?

Good suggestion, [fps target ](https://lmgtfy.app/?q=set+fps+target)is a common term used in various context, but of course we can change it. I think providing the method `setMaxFrameRate(int)` would be enough in that case.

**Comment by wesleycsj** [comment]
Ok. Don't forget to update all examples after in the same commit.

**Review by wesleycsj — APPROVED** [review]
Ok.

**Pull request merged**